src/sweeper/__main__.py

```diff
@@ -63,15 +74,27 @@ def main():
         closet.append(EmptyTest(args['--workspace'], args['--table-name']))
         closet.append(MetadataTest(args['--workspace'], args['--table-name']))
 
-    reports = execute_sweepers(closet, args['--try-fix'])
+    reports = execute_sweepers(closet, args['--try-fix'], args['--change-detect'])
 
     report.print_report(reports)
 
     if args['--save-report']:
         report.save_report(reports, args['--save-report'])
 
+    if args['--scheduled']:
```

Comment:
**question**: do we need two `if args['--scheduled']:` blocks? Is there a reason they aren't merged into one? From looking at the diff it isn't apparent.

Reply:
The first one is basically there to initialize the Supervisor object very early in the process in order to watch over things.  The second one is later and builds the email message based on what's in the "reports" variable after the sweepers have been executed.